v63002/inref/gtm8694.m

```diff
@@ -60,7 +60,8 @@
 zeditfn
 	write "# TESTING ZEDIT",!
 	set $etrap="do incrtrap^incrtrap"
-	zedit "dummy.txt"
+	set dir=$ZDIRECTORY
+	zedit dir_"/dummy.txt"
```

Comment:
Why was this full-path change required?

Reply:
When i recieved the error regarding it not finding the file i included this to try to circumvent it. It did nothing, so I can delete it